### PR TITLE
Fix: float precision for PB_Vector3

### PR DESCRIPTION
**Pull request by caminoezequiel**
# What?
Fixes float precision for PB_Vector3 with `Math.fround`

# Why?
Setting a fractional number in the Transform can lead to unexpected results

Based in: https://github.com/decentraland/explorer/pull/1069

Error:
Screenshot 2020-08-20 at 17 28 53

Fix: 
Screenshot 2020-08-20 at 17 28 53

### CODE REVIEW

File: unity-client/ProjectSettings/QualitySettings.asset [hunk 1]
```diff
@@ -21,7 +21,7 @@ QualitySettings:
     skinWeights: 4
     textureQuality: 0
     anisotropicTextures: 2
-    antiAliasing: 2
+    antiAliasing: 0
```
Comment (pablitar) [inline unity-client/ProjectSettings/QualitySettings.asset]: Was this change voluntary? If not, you may want to revert it :)

**Review by pablitar — APPROVED** [review]
Great work!!

File: unity-client/ProjectSettings/QualitySettings.asset (hunk 1 above)
Comment (caminoezequiel) [reply unity-client/ProjectSettings/QualitySettings.asset]: @pablitar No, It was generated when I ran unity, If it isn't necessary, I'll revert it

**Review by nchamo — APPROVED** [review]
Great work!!!

**Pull request merged**